client/src/docs-ui/page/page.tsx

```diff
@@ -26,7 +26,12 @@ import {
   SetNewSelectedTabHeadings,
 } from "./page.types";
 import {pageContext} from "./page.context";
-import {track, AnalyticsEventType} from "../../utils/track";
+import {
+  track,
```

Comment:
So I see `trackPageVisit()` and `trackPageFetchException()` in use, where is `track` or the `AnalyticsEventType` used?

Reply:
`track` and `AnalyticsEventType` are used right above the `trackPageVisit` and `trackPageFetchException` calls -- they're being used for Pinpoint tracking right now.

Reply:
We'll rip out those `track()` calls once we're fully switched over to Adobe from Pinpoint.  I'm still unsure whether a `switch`-based approach (i.e. calling `track()` with a given event type) or this current approach with the specific named methods like `trackPageVisit` is more appropriate